integration/test-network/README.md

```diff
@@ -1,23 +1,36 @@
 # Setup the FPC test network
 
 The FPC test network builds on the test-network provided by [fabric-samples](https://github.com/hyperledger/fabric-samples).
-
 We provide fabric-samples as a submodule in `$FPC_PATH/integration/test-network/fabric-samples`.
 
-Make sure you have installed [yq](https://github.com/mikefarah/yq).
-Note that you will version v3.4.1 or larger. 
-For Ubuntu, `sudo snap install yq` is the easiest way to get a good version.
+In order to use FPC with the test network, make sure you have installed [yq](https://github.com/mikefarah/yq) (version 3.x).
+Not that newer versions (v4.x and higher) are currently not supported.
+You can install `yq` v3 via `go get`.
+```bash
+GO111MODULE=on go get github.com/mikefarah/yq/v3
+```
+
+In addition to `yq` you need a recent version of docker-compose (version 1.25 or higher).
+Note Ubuntu 18.04 comes with an older version of docker-compose and thus needs to be updated.
+See related notes in [Working from behind a proxy](../../README.md#working-from-behind-a-proxy) in our [README.md](../../README.md) for more information.
+
+[comment]: <> (This comment can be removed with upgrading FPC to general Ubuntu 20.04 support)
+If you run the FPC test network from within our FPC docker dev container, please use Ubuntu 20.04 Docker images.
+```Makefile
+DOCKER_BUILD_OPTS="--build-arg UBUNTU_VERSION=20.04 --build-arg UBUNTU_NAME=focal" make -C $FPC_PATH/utils/docker run
+```
+
```

Comment:
Somehow, above clickable suggestion did not work, so another try.  I also noticed that marcus already mentioned the condition on docker-compose (1.25x or higher). Might be worth though to move to that place from the proxy section the text that 18.04 has an old version and instructions on how to upgrade. 

```suggestion
```